EventFiltering/PWGJE/jetFilter.cxx

```diff
@@ -127,39 +156,30 @@ struct jetFilter {
     spectra.add("hRho", "Underlying event density #rho", HistType::kTH1F,
                 {{200, 0., +20., "#rho (GeV/#it{c})"}});
 
-    auto scalers{std::get<std::shared_ptr<TH1>>(spectra.add(
-      "fProcessedEvents", ";;Number of filtered events", HistType::kTH1F,
-      {{kAllObjects, -0.5, +kAllObjects - 0.5}}))};
-    for (uint32_t iS{1}; iS <= highPtObjectsNames.size(); ++iS) {
-      scalers->GetXaxis()->SetBinLabel(iS, highPtObjectsNames[iS - 1].data());
-    }
+    hProcessedEvents->GetXaxis()->SetBinLabel(kBinAllEvents + 1, "Processed events");
+    hProcessedEvents->GetXaxis()->SetBinLabel(kBinEventsVz + 1, "Processed events with |vz| < 10 cm");
+    hProcessedEvents->GetXaxis()->SetBinLabel(kBinJetChLowPt + 1, "JetChLowPt");
```

Comment:
Hi @fkrizek,
for the automatic checks to work, I will need you to use the name of the column here. You can get them using variations of `o2::aod::filtering::JetChLowPt::columnLabel()`.
Thanks a lot!

Reply:
Hi @mpuccio ,  I have fixed the lables as suggested.